pom.xml

```diff
@@ -202,6 +202,12 @@
             <version>${guava.version}</version>
         </dependency>
         <dependency>
+            <groupId>com.google.auto.value</groupId>
+            <artifactId>auto-value</artifactId>
+            <version>${auto-value.version}</version>
+            <scope>provided</scope>
```

Comment:
provided? Man I actually had to google this one. Why is maven so bloody complicated.